x/delayedack/keeper/rollapp_packet.go

```diff
@@ -107,15 +107,15 @@ func (k *Keeper) UpdateRollappPacketWithStatus(ctx sdk.Context, rollappPacket co
 	// Create a new rollapp packet with the updated status
 	err := k.SetRollappPacket(ctx, rollappPacket)
 	if err != nil {
-		return commontypes.RollappPacket{}, err
+		return rollappPacket, err
```

Comment:
With how we currently setup the test [UpdateRollappPacketWithStatus](https://github.com/dymensionxyz/dymension/blob/933e48305a6b9a59e4212cc36d697936ea57d6e3/x/delayedack/keeper/rollapp_packet_test.go#L183), I think this err is uncatchable. I think we can refactor it later on, wdyt @omritoptix

Reply:
u assert for error, I think it's fine.